File: Finding length of the sequence.js
var lengthOfSequence = function (arr, n) {
  const result_index = arr.map((e,index) => e==n ? index : -1).filter(v => v>=0)
  return result_index.length==2 ? result_index[1]-result_index[0]+1  : 0;
};

File: Finding length of the sequence/Finding length of the sequence.swift
func lengthOfSequence(_ arr:[Int], _ n:Int) -> Int {
  let arrG = arr.enumerated().compactMap { index, element -> Int? in
        return element == n ? index : nil
  }
  if arrG.count != 2 {
      return 0
  }
  return arrG[1] - arrG[0] + 1
}
